crates/apollo-router/src/configuration/otlp/http.rs

```diff
@@ -4,15 +4,15 @@ use opentelemetry_otlp::WithExportConfig;
 use serde::{Deserialize, Serialize};
 use std::collections::HashMap;
 
-#[derive(Debug, Clone, Deserialize, Serialize)]
+#[derive(Debug, Clone, Deserialize, Serialize, Default)]
```

Comment:
> If you use this only for the tests. Use `#[cfg_attr(test, derive(Default))]`.

Yes this is fine. I think it's the same trick for jaeger.

Reply:
this is necessary also when running it with a config file, not only in tests

Reply:
oh sorry I badly copy-pasted stuff. Let me fix my comment you will see it has a totally different meaning XD